provider/aws/sia-ec2/options/options.go

```diff
@@ -267,15 +307,33 @@ func NewOptions(bytes []byte, accountId, metaEndPoint, siaDir, version, ztsCaCer
 		ZTSCACertFile:    ztsCaCert,
 		ZTSServerName:    ztsServerName,
 		ZTSAWSDomain:     ztsAwsDomain,
+		GenerateRoleKey:  generateRoleKey,
+		RotateKey:        rotateKey,
+		BackUpDir:        fmt.Sprintf("%s/backup", siaDir),
+		TaskIdPrefix:     taskIdPrefix,
 	}, nil
 }
 
 func GetOSType() int {
-	// this is only needed if sshd on the given os
-	// needs something special for the restart
-	// typically one would parse the /etc/os-release
-	// for now we have a single method for restarting
-	// sshd so we'll just return unknown os type
-
+	// first check to see if it's centos - currently
```

Comment:
No changes are needed here. CentosOS6 and Ubuntu14 are no longer supported. So the default way of restarting sshd is sufficient for all OS types we support.

Reply:
done